web/src/features/moreCast2/components/MoreCast2Column.tsx

```diff
@@ -180,6 +180,9 @@ export class IndeterminateField implements ColDefGenerator, ForecastColDefGenera
 }
 
 export const tempForecastField = new IndeterminateField('temp', TEMP_HEADER, 'number', 0, true, (value: string) => {
+  if (value === '') {
+    return 'Enter a value'
+  }
```

Comment:
What do you think about extracting this out into a function that can be used across all the forecast fields instead of repeating it?

Reply:
Mixed feelings -- normally I'd agree to dedup, but it's a very small check and adding another level of indirection in this giant monstrosity is has me thinking twice lol.